snuba/web/db_query.py

```diff
@@ -936,7 +936,19 @@ def _apply_allocation_policies_quota(
         stats["quota_allowance"]["summary"] = summary
 
         if not can_run:
+            metrics.increment(
+                "rejected_query",
+            )
```

Comment:
you're not tagging with anything? You should at least tag with dataset or storage key, otherwise this metric is not that useful

Reply:
I'll add the tags, but why isn't a metric like this useful to get a volume % breakdown of successful/rejected/throttled queries?

Reply:
It is useful for that! And if you think about how snuba is used and how queries are throttled, it's all based on the actual data that is being accessed. If you roll up all the throttled queries together then you won't know how many queries were throttled on errors or replays. It will all just be one number and it won't really be helpful. You can still sum across all tags to get the volume % breakdown of course